Examples/drawing-tests.lisp

```diff
@@ -1,100 +1,356 @@
 (in-package :clim-demo)
 
+(defparameter *drawing-tests-categories* nil)
 (defparameter *drawing-tests* (make-hash-table :test 'equal))
 
 (defparameter *width* 500)
 (defparameter *height* 700)
 (defparameter *border-width* 5)
 
-(defstruct drawing-test name description drawer)
+(defstruct drawing-test category name description display-function)
 
-(defmacro define-drawing-test (name arglist description &body body)
+(defun drawing-test-keyname (category name)
+  (concatenate 'string category "-" name))
```

Comment:
is there a need for this to be a string? hashtable access could be better if you had used `alexandria:symbolicate` and have symbols as keys

Reply:
No, you are right. I will change it.